install.sh

```diff
@@ -119,6 +131,7 @@ fi
 mirror=''
 DRY_RUN=${DRY_RUN:-}
 REPO_ONLY=${REPO_ONLY:-0}
+AUTOSTART=${AUTOSTART:-1}
```

Comment:
Perhaps cleaner to change this to `NO_AUTOSTART` to align with the flag, and to be more clear on what's the default.